src/content/en/fundamentals/accessibility/02-semantics-bultin/sembuiltin-four.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: shared/narrow
+title: "The Accessibility Tree"
+description: "Introduction to the Accessibility Tree"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 4
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+Imagine that you're building a user interface *for screen reader users only*. Here, you don't need to create any visual UI at all, but just provide enough information for the screen reader to use. 
+
+What you'd be creating is a kind of API describing the page structure, similar to the DOM API, but you can get away with less information and fewer nodes, because a lot of that information is only useful for visual presentation. It might look something like this. 
+
+![treestructure.png](imgs/treestructure.png)
+
+This is basically what the browser actually presents to the screen reader. The browser takes the DOM tree and modifies it into a form that is useful to assistive technology. We refer to this modified tree as the *Accessibility Tree*.
+
+You might visualize the accessibility tree as looking a bit like an old web page from the '90s: a few images, lots of links, perhaps a field and a button. 
+
+![90swebpage.png](imgs/90swebpage.png)
```

Comment:
This is an image of an old Yahoo page that Alice used in our Udacity course. I'm not sure if we can use it on /web though. Maybe it would be better to use something like this https://upload.wikimedia.org/wikipedia/en/b/b7/Google1998.png